python/pyspark/ml/classification.py

```diff
@@ -223,6 +223,14 @@ def intercept(self):
         """
         return self._call_java("intercept")
 
+    @property
+    @since("1.6.0")
+    def numFeatures(self):
+        """
+        The number of features
```

Comment:
"The number of features" -> "The number of features used to train the model."

I think this is more explicit since, for things like decision trees, this property is what number of features were used in training and not necessarily what number of features the model incorporates in its prediction.

Nit: add period to follow docstring convention.